atef/widgets/config/page.py

```diff
@@ -324,50 +235,50 @@ class PageWidget(QWidget):
     data: AnyDataclass
     parent_button: Optional[QToolButton]
 
-    def __init__(self, data: AnyDataclass, **kwargs):
+    def __init__(
+        self,
+        data: AnyDataclass,
+        tree_item: Optional[TreeItem] = None,
+        full_tree: Optional[DualTree] = None,
+        **kwargs
+    ) -> None:
         super().__init__(**kwargs)
         self.data = data
+        if tree_item is not None:
+            # assign tree item and check
+            assert tree_item.orig_data is data
```

Comment:
Assert statements aren't necessarily run at runtime: https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement, so if this is important it should probably be an if/raise sort of construct

Reply:
Removed this because it is actually wrong.  (PageWidgets can have prepared dataclasses as their `data`)